concepticondata/conceptlists/Clark-2008-441.tsv

```diff
@@ -0,0 +1,442 @@
+Clark-2008-441-211	211	mother’s brother	2692	MATERNAL UNCLE (MOTHER'S BROTHER)
+Clark-2008-441-212	212	mother’s mother	2177	MATERNAL GRANDMOTHER
+Clark-2008-441-213	213	mother	1216	MOTHER
+Clark-2008-441-214	214	mother’s sister	2693	MATERNAL AUNT (MOTHER'S SISTER)
+Clark-2008-441-215	215	o. sister	1758	OLDER SISTER
+Clark-2008-441-216	216	y. sister	1761	YOUNGER SISTER
+Clark-2008-441-217	217	son	1620	SON
+Clark-2008-441-218	218	widow	1756	WIDOW
+Clark-2008-441-219	219	wife	1199	WIFE
+Clark-2008-441-220	220	woman	962	WOMAN
+Clark-2008-441-221	221	artisan		
+Clark-2008-441-222	222	daughter-in-law	1057	DAUGHTER-IN-LAW
+Clark-2008-441-223	223	nung		
+Clark-2008-441-224	224	widower	1981	WIDOWER
+Clark-2008-441-225	225	blouse		
+Clark-2008-441-226	226	bracelet	1754	BRACELET
+Clark-2008-441-227	227	cloth	1616	CLOTH
+Clark-2008-441-228	228	clothing	1895	CLOTHES
+Clark-2008-441-229	229	comb	1475	COMB
+Clark-2008-441-230	230	earring	770	EARRING
+Clark-2008-441-231	231	needle	1382	NEEDLE (FOR SEWING)
+Clark-2008-441-232	232	ring	1374	RING
+Clark-2008-441-233	233	shirt/tunic	1622	SHIRT
+Clark-2008-441-234	234	skirt	1600	SKIRT
+Clark-2008-441-235	235	thread	1161	THREAD
+Clark-2008-441-236	236	trousers	809	TROUSERS
+Clark-2008-441-237	237	awl	278	AWL
+Clark-2008-441-238	238	axe	677	AXE
+Clark-2008-441-239	239	bag	2468	BAG
+Clark-2008-441-240	240	basket i	1539	BASKET
+Clark-2008-441-241	241	basket ri		
+Clark-2008-441-242	242	basket wi		
+Clark-2008-441-243	243	bowl	1530	BOWL
+Clark-2008-441-244	244	chopsticks	1987	CHOPSTICKS
+Clark-2008-441-245	245	fire	221	FIRE
+Clark-2008-441-246	246	fire tongs	272	TONGS
+Clark-2008-441-247	247	fishnet	388	FISHNET
+Clark-2008-441-248	248	hammer	1481	HAMMER
+Clark-2008-441-249	249	harrow	2545	HARROW (TOOL)
+Clark-2008-441-250	250	hoe	284	HOE
+Clark-2008-441-251	251	knife	1352	KNIFE
+Clark-2008-441-252	252	masher		
+Clark-2008-441-253	253	plane	1597	AIRPLANE
+Clark-2008-441-254	254	plow	2154	PLOUGH (INSTRUMENT)
+Clark-2008-441-255	255	carrying pole	3432	SHOULDER POLE
+Clark-2008-441-256	256	rope	1218	ROPE
+Clark-2008-441-257	257	sickle	341	SICKLE
+Clark-2008-441-258	258	anvil	1648	ANVIL
+Clark-2008-441-259	259	chisel	1979	CHISEL
+Clark-2008-441-260	260	peg	361	PEG
+Clark-2008-441-261	261	pot	1462	POT
+Clark-2008-441-262	262	ashes	646	ASH
+Clark-2008-441-263	263	bed	1663	BED
+Clark-2008-441-264	264	blanket	806	BLANKET
+Clark-2008-441-265	265	boat	1844	BOAT
+Clark-2008-441-266	266	bridge	1011	BRIDGE
+Clark-2008-441-267	267	charcoal	1188	CHARCOAL
+Clark-2008-441-268	268	coop		
+Clark-2008-441-269	269	dock, boat		
+Clark-2008-441-270	270	door	1567	DOOR
+Clark-2008-441-271	271	firewood	10	FIREWOOD
+Clark-2008-441-272	272	garden	586	GARDEN
+Clark-2008-441-273	273	grave	1773	GRAVE
+Clark-2008-441-274	274	house	1252	HOUSE
+Clark-2008-441-275	275	mat	195	MAT
+Clark-2008-441-276	276	paddy	140	PADDY FIELD
+Clark-2008-441-277	277	pillow	1044	PILLOW
+Clark-2008-441-278	278	plot dry		
+Clark-2008-441-279	279	wet field	140	PADDY FIELD
```

Comment:
If the word _paddy_ is mapped to PADDY FIELD, I'd suggest to delete the mapping to _wet field_. Except the glosses are identical.